other/concat-exports.js

```diff
@@ -0,0 +1,45 @@
+const appendToFile = require('fs').appendFileSync
+const resolve = require('path').resolve
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+function capitalize(s) {
+  return s.slice(0, 1).toUpperCase() + s.slice(1)
+}
+
+function dashToCamelCase(s) {
+  return s.replace(/-([a-z])/g, ([, char]) => char.toUpperCase())
+}
+
+function isCapitalized(s) {
+  return !!s && s[0] === s[0].toUpperCase()
+}
+
+// node version <= 7 does not support trailing commas in function calls
+// prettier-ignore
+const jsBuiltInConstructorNames = Object.getOwnPropertyNames(global)
+  .filter(isCapitalized)
```

Comment:
I'm more concerned about readability/maintainability of this file than performance and would prefer to not include anything in here that's only here for performance reasons. The script is really fast as it is :)